### PR TITLE
Reclaims the Civilian Account for the Club

### PR DESCRIPTION
**Pull request by zombiemimic**
## About The Pull Request
Gives the Civilian Account to the club and adds a new account for Offship vendors

## Why It's Good For The Game
Gives a fund so the club is not caught with its pants down when starting

## Changelog
:cl:
add: Added Offship account
tweak: Changed around some vendor accounts
/:cl:

### CODE REVIEW

File: code/game/jobs/department.dm [hunk 1]
```diff
@@ -126,3 +125,7 @@
 	account_initial_balance = 7500
 	funding_type = FUNDING_NONE
 
+/datum/department/offship //So we can pay the Club without giving them independant money
+	name = "Offship entities"
+	id = DEPARTMENT_OFFSHIP
+	account_budget = 0
```
Comment (Iglpigl) [inline code/game/jobs/department.dm:131]: No newline.

**Review by Iglpigl — CHANGES_REQUESTED** [review]

File: code/game/jobs/department.dm (hunk 1 above)
Comment (zombiemimic) [reply code/game/jobs/department.dm:131]: Will take care of in the morning

**Comment by Firefox13** [comment]
Where the snack vendors are at? You understand that we've removed them all entirely? Putting them back even with club accounts linked will only aggravate the issue of it not being visited.

**Comment by Sparrowbird611** [comment]
You should put a single one in the club IMO, that way people know the club always has food.

**Review by Firefox13 — APPROVED** [review]

**Pull request merged**